### PR TITLE
Create fallback for mobile devices for ws-multiple-input 

### PR DESCRIPTION
**Pull request by johnsintal**
Please see discussion https://wealthsimple.atlassian.net/browse/ONBOARD-471 on the issues we've experienced with multiple input. 

Tom, the designer, has recommended to fallback using a single input for mobile browsers, this will include tablets (in the spirit of Progressive enhancement)

## Demo 1 : Chrome in normal desktop mode.
screen shot 2018-01-02 at 3 30 34 pm

## DEMO 2: Chrome in browser mode.
screen shot 2018-01-02 at 3 30 59 pm

### CODE REVIEW

File: src/multiple-input/ws-multiple-input.tmpl.html [hunk 1]
```diff
@@ -4,9 +4,21 @@
...
     ng-readonly="$ctrl.isReadonly" />
+
+  <div class="input-group" ng-if="$ctrl.singleInputMode">
+    <input
+            type="text"
+            class="form-input-text  size-sm"
+            maxlength="{{::$ctrl.inputCount}}"
+            ng-model="$ctrl.singleInput"
+            ng-readonly="$ctrl.isReadonly" />
+    <button ng-click="$ctrl.submit()" class="button primary-action">Submit</button>
```
Comment (johnsintal) [inline src/multiple-input/ws-multiple-input.tmpl.html]: Whoops! Forgot to make the locale friendly. I hope there's a precedent case on this or we can just pass the text from the consumer on the widget.

Comment (DanGodfrey) [reply src/multiple-input/ws-multiple-input.tmpl.html]: Good catch. Translations should already be wired up in this repo: https://github.com/wealthsimple/ws-directives/blob/master/src/translations/locale-en_CA.json

**Comment by johnsintal** [comment]
@trmcami @f3ndot I would like to reproduce this in my local env, however, i can't get to 2fa in local. AuthService throws a 500. Perhaps one of you could help me with testing this locally.

Also, @trmcami I would like to see how you did your mobile testing in browser stack. I've use Chrome emulation which does a pretty good job, but nothing beats real hardware.

**Comment by johnsintal** [comment]
screen shot 2018-01-05 at 3 11 36 pm
ignore small res

**Comment by f3ndot** [comment]
@johnsintal you can rely on AuthService staging if you want, but neccessitates you create a staging user first and then update your local user's slug to match the staging one

**Pull request merged**